### ISSUE TITLE
docker container names on grafana and prometheus

### ISSUE THREAD

**Issue by shankar-bala** [firehol/netdata#3736]
Hi,

I am new to netdata and trying to expose the  metrics to prometheus..i want to monitor cpu and memory usage per containers..

i see metric names - netdata_cgroup_cpu_percent_average and netdata_cgroup_mem_usage_MB_average..if i see it in prometheus., i see the chart names, dimension etc..

how i extract the container names so i can plot a graph in grafana..as of now., i see only chart names as "cgroup_<container_name>.cpu".

basically i want to plot memory and cpu usage per containers and should display the container names.

**Comment by ktsaou** [comment]
I am sorry, I don't know. This is a Grafana and Prometheus question. I have the impression they have some kind of pattern matching capabilities that allow you to match the container name.

**Comment by ktsaou** [comment]
Closing due to inactivity.
If the issue remains, feel free to post here.
